### PR TITLE
Working version with tests

### PR DESCRIPTION
**Pull request by josh-dadswell-hackney**

### CODE REVIEW

File: RepairsApi/Handler.cs [hunk 1]
```diff
@@ -1,58 +1,83 @@
...
+    }
 
-        public static async Task SyncDRSWorkOrders()
+
+    public static class ConfigureServices
+    {
+        public static void Configure(this IServiceCollection services)
         {
-            var webHost = BuildWebHost(null);
+            var connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING");
```
Comment (Asura5) [inline RepairsApi/Handler.cs:64]: Could include a null check here to handle if the variable cannot be found

File: RepairsApi/aws-lambda-tools-defaults.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (Asura5) [inline RepairsApi/aws-lambda-tools-defaults.json:1]: Is this file included to help others run the lambda locally?

Comment (josh-dadswell-hackney) [reply RepairsApi/aws-lambda-tools-defaults.json:1]: yes

File: RepairsApi/Handler.cs (hunk 1 above)
Comment (josh-dadswell-hackney) [reply RepairsApi/Handler.cs:64]: This is unused I just forgot to remove it.

**Review by Asura5 — APPROVED** [review]

**Pull request merged**